recipes/termcap/all/conanfile.py

```diff
@@ -72,9 +77,16 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
 
+    @property
+    def _termcap_path(self):
+        return os.path.join(self.package_folder, "bin", "etc", "termcap")
+
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "Termcap"
         self.cpp_info.names["cmake_find_package_multi"] = "Termcap"
         self.cpp_info.libs = tools.collect_libs(self)
         if self.options.shared:
             self.cpp_info.definitions = ["TERMCAP_SHARED"]
+
+        self.output.info("Setting TERMCAP environment variable: {}".format(self._termcap_path))
+        # self.env_info.TERMCAP = self._termcap_path
```

Comment:
```suggestion
        self.env_info.TERMCAP = self._termcap_path
```

Reply:
Oops, forgot to re-enable this after checking whether disabling it would make the errors re-appear :D

Reply:
Okay, so should it be enabled or not? If not, I would prefer removing the output

Reply:
It must be set.
Otherwise the error `tgetent: database not found` will be shown.

It isn't required when running on the build system, but is when creating a portable package.